src/org/traccar/api/MediaFilter.java

```diff
@@ -54,34 +54,32 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 }
             }
             if (userId == null) {
-                throw new NotAuthorizedException("Not authorized");
+                ((HttpServletResponse) response).setStatus(HttpServletResponse.SC_UNAUTHORIZED);
+                response.getWriter().println("Not authorized");
```

Comment:
Do we really need to write some text in the output? I think it makes sense for exception only. Hardcoded text doesn't provide any value over status code.